src/sql/pg/create.sql

```diff
@@ -1,7 +1,7 @@
 create table if not exists "accounts" (
 "id" serial not null primary key,
 "name" varchar(255),
-"balance" float check (balance >= minimum_allowed_balance),
+"balance" float default 0 check (balance >= minimum_allowed_balance),
```

Comment:
Yeah, I'd like to, but I'm getting some weird behavior when inserting multiple rows without balances. For some reason knex fills in the missing balance with `undefined` and the db throws a not null constraint, rather than filling in with the default value 0. This doesn't seem to be a problem when inserting one record at a time. Still looking into the best way to handle this.